inc/problem.class.php

```diff
@@ -304,257 +304,6 @@ public static function pdfMain(PluginPdfSimplePDF $pdf, Problem $job)
         $pdf->displaySpace();
     }
 
-    public static function pdfForItem(PluginPdfSimplePDF $pdf, CommonDBTM $item, $tree = false)
```

Comment:
never called ?

Reply:
I didn't find anything, but I'd like you to check if I've missed anything?!

Reply:
I don't see a call actually, this function seems to be used in PHP objects representing relationship tables. 
so it's fine for me